cves/CVE-2012-0848.yml

```diff
@@ -90,7 +94,7 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 5257743aee0c3982f0079e6553aabc6aa39401d2
-  note: ''
+  note: 'Simple math fix. Puts the output buffer to the correct location.'
 - commit: ''
   note: ''
 vccs:
```

Comment:
Maybe mention the differences in the vccs, and mention which one was more likely to contribute to the vulnerability

Reply:
Thanks. I did this in my fix below. This helped me.